app/assets/stylesheets/galleries.scss

```diff
@@ -73,3 +73,20 @@
   }
 }
 .loading-icon { width: 16px; height: 16px; }
+
+/* Various tables with icons in previously styled with .green */
+.icons-box {
+  background-color: #B0BBA9;
+}
+
+/* Icon-related text previously styled with .green */
+.icon-text {
+  background-color: #B0BBA9;
+}
+
+/* The specific icon boxes on replace icon and replace character */
```

Comment:
The small icon boxes? I think these are styled semantically similarly to, say, the icon box on a post?

Reply:
I /can/ but then I would have to put the specified padding and stuff back too.

Reply:
Hm, okay, I would keep them separate, then. Same thing as I mentioned elsewhere for `!important`, though – if you can remove that and it doesn't change the UI, I would much appreciate that. (And the comment comment (ha) still stands.)

Reply:
did the comment change and the `!important` thing